### PR TITLE
Update advanced-hunting-fileprofile-function.md

### PR DESCRIPTION
**Pull request by amirsc3**
Adjusting explanation to avoid customer confusion. Based on dev confirmation - this will return false if the binary is not built-in to Windows OS.

### CODE REVIEW

File: microsoft-365/security/defender/advanced-hunting-fileprofile-function.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ The `FileProfile()` function is an enrichment function in [advanced hunting](adv
 | `Issuer` | `string` | Information about the issuing certificate authority (CA) |
 | `SignerHash` | `string` | Unique hash value identifying the signer |
 | `IsCertificateValid` | `boolean` | Whether the certificate used to sign the file is valid |
-| `IsRootSignerMicrosoft` | `boolean` | Indicates whether the signer of the root certificate is Microsoft |
+| `IsRootSignerMicrosoft` | `boolean` | Indicates whether the signer of the root certificate is Microsoft and file is built-in to Windows OS |
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/advanced-hunting-fileprofile-function.md]:
```suggestion
| `IsRootSignerMicrosoft` | `boolean` | Indicates whether the signer of the root certificate is Microsoft and the file is built-in to Windows OS |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @amirsc3, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@schmurky Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@schmurky Gentle reminder. Thanks

**Pull request merged**